it/src/test/java/apoc/it/core/StartupTest.java

```diff
@@ -64,6 +64,8 @@ public void check_basic_deployment() {
                 assertTrue(functionCount > 0);
                 assertTrue(coreCount > 0);
                 // Check there's one and only one logger for apoc inside the container
+                // and it doesn't override the one inside the database
+                assertFalse(startupLog.contains("[main] INFO org.eclipse.jetty.server.Server"));
```

Comment:
Looking on the two log output versions in the descriptions of this PR, both contains this?